src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -125,9 +125,18 @@
 
 						if ( $this.tribe_has_attr( 'data-tribejson' ) ) {
 
-							var data = $this.data( 'tribejson' );
+							var data = $this.data( 'tribejson' )
```

Comment:
Still ought to have the terminating `;` ... even though it's not strictly needed, that's our style/is what our coding standards expect